### ISSUE TITLE
Changing screen orientation?

### ISSUE THREAD

**Issue by shekit** [UCTRONICS/UCTRONICS_LCD35_RPI#2]
How can the screen orientation be changed (such as landscape/portrait)? I tried editing /boot/config.txt as well as config.txt inside the uctronics folder but none seem to impact the screen

**Comment by UCTRONICS** [comment]
@shektit

Hi,
The screen can't support rotation.

Regards.

**Comment by UCTRONICS** [comment]
@shekit
Now ,you can rotation it to 180 by running sudo ./UCTRONICS_LCD35_rotation_180_install

**Comment by JamesColinClark** [comment]
I have attempted to command to rotate the screen 180 but I get the command not found.  I installed the UCTRONICS LCD35 using the first three commands on the page:
https://github.com/uctronics/uctronics_lcd35_rpi/

I installed the touchscreen calibration and the virtual keyboard.  Then I went to step NO.4 to rotate the keyboard using the following steps:
cd UCTRONICS_LCD35_RPI
(got into the directory)

sudo chmod 777 UCTRONICS_LCD35_rotation_180_install
(No such file or directory)
... And therefore I cannot execute the rotation...

sudo ./UCTRONICS_LCD35_rotation_180_install

So what steps do I need to take to rotate the screen?
Bear in mind that I am new to Raspberry pi as of today.

**Comment by JamesColinClark** [comment]
*UPDATE*
I was able to rotate the screen.  I had a spelling error on the sudo chmod 777 command.
FYI, using Raspberry Pi 3 NOOBS

**Comment by zlyzen** [comment]
Screen works well (and touchscreen works correctly) with no rotation. 

However, after executing: sudo ./UCTRONICS_LCD35_rotation_180_install

touchscreen functionality is lost and when I run calibration program I get the message 
"Error:No calibratable devices found"

Returning to no rotation with sudo ./UCTRONICS_LCD35_install brings back touchscreen functionality.

**Comment by MetalicLynx** [comment]
Yep, and agreed, as long as you just leave it as the basic install it works fine.
HOWEVER, IT IS INCAPABLE OF ROTATING!!!!!!!!!!
If you want it to be part of an art project, you can use it, but all your images will need to be rotated to make up for the lack of ability of this screen.  Really, no civic pride.
